ding/config/config.py

```diff
@@ -380,12 +364,12 @@ def compile_config(
             env_config = env.default_config()
         else:
             env_config = EasyDict()  # env does not have default_config
-        env_config = deep_merge_dicts(env_config_template, env_config)
+        env_config = deep_merge_dicts(env.env_config_template, env_config)
```

Comment:
How we deal with the relationship of `policy.policy_config_template` and `policy.default_config()`, it would lead to some confusion.